rewrite-java/src/main/java/org/openrewrite/java/internal/template/BlockStatementTemplateGenerator.java

```diff
@@ -250,8 +250,14 @@ private void template(Cursor cursor, J prior, StringBuilder before, StringBuilde
             before.insert(0, "assert ");
         } else if (j instanceof J.NewArray) {
```

Comment:
This case was really specific to the handling of expressions in an array's initializer. This is still required, but when replacing the `J.NewArray` as a whole, we use the same logic as for other expressions (i.e. using `__M__.any()`).